dolphinscheduler-python/pydolphinscheduler/docs/source/tutorial.rst

```diff
@@ -219,5 +219,155 @@ named "tutorial" or "tutorial_decorator". The task graph of workflow like below:
    :language: text
    :lines: 24-28
 
+Create Process Using YAML File
+------------------------------
+
+We can use pydolphinscheduler CLI to create process using YAML file
+
+.. code-block:: bash
+
```

Comment:
I think we should only add a tutorial exampel instead of all yaml in this file. you can see more detail in 

```rst
.. tab:: Tradition
.. tab:: Task Decorator
```

in this file. Maybe you can add new `tab` named `Yaml File` or others.

And then for the detail for the detail of task type like `shell`, `SQL` I prefer to add they in seperate directory, Or maybe we can add them to exists tasks dirrectory, like yaml shell into https://dolphinscheduler.apache.org/python/dev/tasks/shell.html WDYT

Reply:
good idea, I will change it

Reply:
done